enterprise/coderd/dormancy/dormantusersjob.go

```diff
@@ -57,8 +60,29 @@ func CheckInactiveUsersWithOptions(ctx context.Context, logger slog.Logger, db d
 				continue
 			}
 
+			af := map[string]string{
+				"automatic_actor":     "coder",
+				"automatic_subsystem": "dormancy",
```

Comment:
this may be outside the scope of this PR but seems like these should be defined somewhere. This is the first time I'm seeing them and it's not clear when we should be passing them as extra fields.

Reply:
Yeah, I'll make a constant for it